snippets/distr-tracing-tempo-tempostack-custom-resource.adoc

```diff
@@ -4,40 +4,49 @@
 // * modules/distr-tracing-tempo-install-tempostack-web-console.adoc
 // * modules/distr-tracing-tempo-install-tempostack-cli.adoc
 
+.Example `TempoStack` CR for AWS S3 and MinIO storage and two tenants
 [source,yaml]
 ----
 apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoStack
+kind: TempoStack # <1>
 metadata:
-  name: sample
+  name: simplest
   namespace: <project_of_tempostack_instance>
```

Comment:
@rubenvp8510 
Or should this be the namespace?
```suggestion
  namespace: chainsaw-multitenancy
```

Reply:
Well, it is a placeholder for whatever namepace you want to deploy the solution. May be we should clarify it with the numbers? Like 
`
  namespace: <project_of_tempostack_instance> <1> `

Reply:
Thank you for the clarification and suggestion. Adding the callout as you suggested.